package.json

```diff
@@ -87,24 +86,22 @@
 		"preact": ">=10"
 	},
 	"devDependencies": {
-		"babel-plugin-transform-object-rest-spread": "^6.26.0",
-		"babel-plugin-transform-react-jsx": "^6.24.1",
-		"babel-preset-env": "^1.7.0",
-		"babel-register": "^6.26.0",
-		"chai": "^3.5.0",
-		"copyfiles": "^1.2.0",
-		"eslint": "^4.19.1",
-		"eslint-config-developit": "^1.1.1",
-		"husky": "^4.3.0",
-		"lint-staged": "^10.4.0",
-		"microbundle": "^0.6.0",
-		"mocha": "^5.2.0",
-		"npm-merge-driver-install": "^1.1.1",
```

Comment:
I removed npm-merge-driver-install since it seems the underlying npm-merge-driver is unmaintained and is responsible for weekly emails from GitHub in my inbox.

npm now auto merges package-lock.json conflicts by just running `npm i`. So my workflow now when I have a conflict is to first, fix up in conflicts in package.json then run `npm i` and npm won't fail.